pkg/util/controller_utils.go

```diff
@@ -101,3 +103,38 @@ func GetNodeCondition(status *v1.NodeStatus, conditionType v1.NodeConditionType)
 	}
 	return -1, nil
 }
+
+// CreateAddNetworkHandler creates an add network handler.
+func CreateAddNetworkHandler(f func(network *networkv1.Network) error) func(obj interface{}) {
```

Comment:
better not to add this helpers, it doesn't add much value since are only used once and makes the code harder to read, since you have to scroll to different places, and complicate the code with closures

Reply:
tested, and controller will not crash, the factory will keep sending error msg that the specific object is not available. BTW, it already does that Today for GNP.

Reply:
Does this error message keep repeating? We should try to control the frequency if this happens.

Reply:
yes it does. I looked into make MN parameterized, but it is not straight forward, so better course of action will be to always install the CRDs.

Reply:
Makes sense. I will make this change.